pyproject.toml

```
@@ -28,6 +28,28 @@ dependencies = [
     "shapely >= 2.0.0",
 ]
 
+[project.optional-dependencies]
+geodatabase = [
+    "psycopg2-binary>=2.8.0",
+    "SQLAlchemy>=1.3",
+]
+geocoding = [
+    "geopy",
+]
+plotting = [
+    "matplotlib>=3.5.0",
+    "mapclassify",
+    "cartopy",
+    "geoplot",
```

```suggestion
    "xyzservices",
```
geoplot is no longer maintained and should be removed, and I think it makes sense to include xyzservices here.  

I also think cartopy probably should not be here since we don't actually have any dependence on it - it's just part of the ecosystem.